src/contracts/atlas/Escrow.sol

```diff
@@ -268,18 +269,6 @@ abstract contract Escrow is AtlETH {
             return (1 << uint256(SolverOutcome.UserOutOfGas), gasLimit); // gasLimit = 0
         }
 
-        if (solverOp.deadline != 0 && block.number > solverOp.deadline) {
-            return (
-                1
-                    << uint256(
-                        dConfig.callConfig.allowsTrustedOpHash()
-                            ? uint256(SolverOutcome.DeadlinePassedAlt)
-                            : uint256(SolverOutcome.DeadlinePassed)
-                    ),
-                gasLimit // gasLimit = 0
-            );
-        }
-
         gasLimit = _SOLVER_GAS_LIMIT_SCALE
             * (solverOp.gas < dConfig.solverGasLimit ? solverOp.gas : dConfig.solverGasLimit)
             / (_SOLVER_GAS_LIMIT_SCALE + _SOLVER_GAS_LIMIT_BUFFER_PERCENTAGE) + _FASTLANE_GAS_BUFFER;
```

Comment:
The `_verifySolverEligibility()` gas limit calc is different. I think it should be this as well. If using a lib to share code between sorter and here doesn't increase gas or size, I think thats a good idea too.

Reply:
changed sorter calc to replicate this

Reply:
Fixed here https://github.com/FastLane-Labs/atlas/pull/276/commits/9cdb1dcaa65e7c808360080e7fb1ef51e8fde3ad